event_files/speculation_metrics.json

```diff
@@ -0,0 +1,226 @@
+[
+    {
+        "MetricName": "control_flow_pki",
+        "BriefDescription": "The rate of control flow instructions retired per kilo instructions",
+        "MetricExpr": "SPEC.CONTROL_FLOW.RETIRED / GEN.INST_RETIRED * 1000",
+        "MetricGroup": "Speculation",
+        "ScaleUnit": "1PKI"
+    },
+    {
+        "MetricName": "control_flow_mpki",
+        "BriefDescription": "The rate of control flow instructions mis-predicted per kilo instructions",
+        "MetricExpr": "SPEC.CONTROL_FLOW.MISPRED / GEN.INST_RETIRED * 1000",
+        "MetricGroup": "Speculation",
+        "ScaleUnit": "1MPKI"
```

Comment:
the "M" in "1MPKI" stands for mis-...? or sth else? Why it's not 1PKI? or just "PKI"?

Reply:
The 'M' here stands for 'Mispredict'. So MPKI should read as 'Mispredicts-Per-Kilo-Instructions'.

Reply:
FWIW, I think MPKI is a pretty standard and commonly used abbreviation.  Though I see it used for both mispredicts PKI (w.r.t. branch prediction) and misses PKI (w.r.t. cache accesses).

Reply:
Yep, the 'MPKI' is somewhat esablished terminology and we are trying to follow it here.